templates/profiles.html

```diff
@@ -299,9 +299,9 @@ <h3>Create a profile</h3>
                         </div>
                         <div class="field-body">
                             <div class="field">
-                                <div class="control">
+                                <div class="control auto-complete">
```

Comment:
The autocomplete on the search works great here! I'd add an autocomplete to the tactic, technique Id and name just like I suggested for the abilities page